frontend/src/Components/ProductCard.js

```diff
@@ -0,0 +1,64 @@
+import ReactTooltip from "react-tooltip";
+import { Link } from "react-router-dom";
+
+const ProductCard = ({
+  productDetails: {
+    _id,
+    name,
+    price,
+    image,
+    rating,
+    description,
+    category,
+    brand,
+  },
+}) => {
+  const tooltipContent = `
+    <ul>
+    <li><strong>Brand: </strong>${brand}</li>
+    <li><strong>Category: </strong>${category}</li>
+    <li><strong>Rating: </strong>${rating}</li>
+    </ul>
+  `;
+
+  return (
+    <>
+      <div className="card vertical">
+        <div
+          data-tip={tooltipContent}
```

Comment:
Appreciation : This feature was something new and looked so cool to look at in the UI !